docs/features/service_client_constructors.md

```diff
@@ -0,0 +1,56 @@
+## Service Client Constrctors
+
+Scope: This document only applies to DPG services.
+
+There will be two **public** client constructors in a generated service client. One primary public constructor with all the user defined client parameters plus `ClientOptions`, and one secondary public constructor with all the user defined **required** client parameters. The secondary public constructor calls the primary public constructor underneath.
```

Comment:
```suggestion
There will be two **public** client constructors in a generated service client. One primary public constructor with all the user defined client parameters plus a `ClientOptions` parameter, and one secondary public constructor with all the user defined **required** client parameters without the `ClientOptions` parameter. The secondary public constructor calls the primary public constructor underneath with default values of those optional parameters.
```

Reply:
This section has some problem. I adjust a bit.